groups/instance/cloud-init/templates/bootstrap-commands.yml.tpl

```diff
@@ -0,0 +1,6 @@
+runcmd:
+  - systemctl enable artifactory
+  - wait
+  - cp /opt/jfrog/artifactory/var/etc/artifactory/artifactory.config.import.xml /opt/jfrog/artifactory/var/etc/artifactory/artifactory.config.import.xml.original
+  - wait
+  - systemctl restart artifactory
```

Comment:
Not sure why what looks to be a config backup is being made here. Surely we should be deploying a correctly configured file?

Also, is the use of `wait` commands necessary here? The commands should be structured in a way where enforcing wait states is not required.